src/chain/chain_accumulator.rs

```diff
@@ -31,6 +68,11 @@ impl ChainAccumulator {
         proof_set: ProofSet,
     ) -> Result<(), InsertError> {
         if self.completed_events.contains(&event) {
+            let voters: Vec<_> = proof_set.ids().cloned().collect();
+            let _ = self
+                .vote_statuses
+                .iter_mut()
+                .any(|vote_status| vote_status.add_votes(&event, &voters));
```

Comment:
Will ` vote_status.add_votes(&event, &voters)` ever do anything in this position?

Reply:
if it's a completed event, then should do something?
unless you mean this insert_with_proof_set is a subset call after a chain.poll_event, i.e. they contain the same proof_set?

Reply:
How could that be a completed event and this function still be called (this is only for Supermarjority consensus event i.e Add/Remove)

Reply:
Ah, I got it. So we shall not track it, right? as the Supermarjority consensus won't have ALL proofs.